sys-devel/gcc/gcc-5.4.0_2016_07_08.recipe

```diff
@@ -0,0 +1,347 @@
+SUMMARY="C/C++ compiler for target ${effectiveTargetMachineTriple}"
+DESCRIPTION="The standard compiler for non-legacy Haiku (i.e. for all \
+architectures other than x86_gcc2)."
+HOMEPAGE="http://gcc.gnu.org"
+
+SOURCE_URI="https://ftp.gnu.org/gnu/gcc/gcc-5.4.0/gcc-5.4.0.tar.bz2"
```

Comment:
* Do we have any rules that forbid to use a redirector that would not use https?
* Are we allowed to say that some existing (or implicit) rule is wrong and change it?
* Who are we to force GNU to request that their mirrors accept https?
* If everybody refused to use download redirector services just because they use http instead of https, then most mirrors would be useless.
* What's the value added of using https if we already have a safe way to check that the archives are OK?
* Even if the majority thinks that we should use https for SOURCE_URI, this does not mean it is right.

I'm not going to stop contributing if you decide that we should not use "unsafe" SOURCE_URI, but I really think the project should avoid that mistake. If I did not manage to convince you, would you agree to ask others what they think about this question?
I'm not trying to fight against you. And I wish I had found a better way to explain my point of view.
Last but not least: I won't **ever** start saying "ha ha, I won!" if other contributors say I'm right.